### ISSUE TITLE
project-2 can not run anchor test command

### ISSUE THREAD

**Issue by nakedkoalacapital** [solana-developers/developer-bootcamp-2024#4]
```
found 0 vulnerabilities
root@Ubuntu-2204-jammy-amd64-base ~/workspace/developer-bootcamp-2024/project-2-voting/anchor # anchor test --skip-local-validator --skip-deploy
    Finished release [optimized] target(s) in 0.32s
    Finished `test` profile [unoptimized + debuginfo] target(s) in 0.48s
     Running unittests src/lib.rs (/root/workspace/developer-bootcamp-2024/project-2-voting/anchor/target/debug/deps/voting-8f27fe363d7d4c45)

Found a 'test' script in the Anchor.toml. Running it as a test suite!

Running test suite: "/root/workspace/developer-bootcamp-2024/project-2-voting/anchor/Anchor.toml"


node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^
{
  stack: "TypeError: Cannot read properties of null (reading 'message')\n" +
    '    at handleWorkspaceChanges (/root/workspace/developer-bootcamp-2024/project-2-voting/node_modules/nx/src/daemon/server/server.js:203:90)\n' +
    '    at /root/workspace/developer-bootcamp-2024/project-2-voting/node_modules/nx/src/daemon/server/watcher.js:36:9',
  message: "Cannot read properties of null (reading 'message')"
}

Node.js v19.0.0

```

**Comment by jacobcreech** [comment]
Could you try with anchor-bankrun@4.0?

**Comment by jacobcreech** [comment]
Solved with anchor-bankrun@5.0
